deploy/yats-1.0/debian/copyright

```diff
@@ -0,0 +1,17 @@
+ 
+This package was debianised by Mathias Flueggen <mathias.flueggen@student.hpi.de> on Wed, 04 Apr 2018 17:16:54 +0200
+
+It was downloaded from:
+
+    https://github.com/hpicgs/yats
+    
+Upstream Author:
+
+    Lawrence Benson <lawrence.benson@student.hpi.de>
+    Jan van Dieken <jan.dieken@student.hpi.de>
+    Mathias Flueggen <mathias.flueggen@student.hpi.de>
+    Konstantin Harmuth <konstantin.harmuth@student.hpi.de>
```

Comment:
Is it required to write the email here?

Reply:
maybe just the github name?

Reply:
Willy had it in his example. I'll just put "YaTS Team"